src/app/shared/components/message-bar/message-bar.component.spec.ts

```diff
@@ -1,29 +1,76 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
 import { MatIconModule } from '@angular/material/icon';
-import { MatSnackBarModule, MAT_SNACK_BAR_DATA } from '@angular/material/snack-bar';
+import { MatSnackBarModule, MAT_SNACK_BAR_DATA, MatSnackBar } from '@angular/material/snack-bar';
 import { TranslateModule } from '@ngx-translate/core';
 
 import { MessageBarComponent } from './message-bar.component';
+import { MessageBar } from 'shared/models/messageBar.model';
 
 describe('SnackBarComponent', () => {
   let component: MessageBarComponent;
   let fixture: ComponentFixture<MessageBarComponent>;
+  let matSnackBar: MatSnackBar;
+  let mockData: MessageBar;
+
+  beforeEach(() => {
```

Comment:
We already have a `beforeEach` block, you can move it before `TestBed.configureTestingModule` in the same block.